README.md

```diff
@@ -48,6 +48,8 @@ If the basic registers change you can restart your register so that it sees the
 - Install and run postgres db
 - Checkout project
 - Run the `./go` script to set everything up
+- `$ git submodule init`
+- `$ git submodule update`
```

Comment:
I don't think we want to have `git submodule update` in the `go` script since this will update the submodules which could have adverse effects to the UI